src/popup/pages/Send/index.js

```diff
@@ -184,9 +184,13 @@ class SendPage extends React.Component {
             toAddressCanInputDefaultValue = currentAccount.address
             sendAction = WALLET_SEND_RUNTIME_WITHDRAW
           }
-          // A wild guess: the minimum gas price on Emerald (100 nano ROSE) times the default loose
-          // overestimate of the gas (15k).
-          defaultFeeAmount = "1500000"
+          // TODO: make it paratime specific: Emerald 100 nano ROSE, Cipher 20
```

Comment:
Cipher is actually 5 now.

Reply:
I changed the comment to 5, but if I lower the overestimation, it will break Emerald (until we make it paratime specific)